### PR TITLE
Add Anaconda CE kustomize templates

### PR DESCRIPTION
**Pull request by Xaenalt**

### CODE REVIEW

File: partners/anaconda/base/anaconda-ce-imagestream.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: s2i-minimal-notebook-anaconda
+  labels:
+    opendatahub.io/notebook-image: "false"
+    opendatahub.io/modified: "false"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/Xaenalt/s2i-minimal-notebook"
```
Comment (cfchase) [inline partners/anaconda/base/anaconda-ce-imagestream.yaml:9]: should this be your private github or should it be in an official org somewhere?

File: partners/anaconda/base/anaconda-ce-validator-cron.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: anaconda-ce-periodic-validator
+  namespace: redhat-ods-applications
+  labels:
+    opendatahub.io/modified: "false"
+spec:
+  schedule: "0 0 * * *"
```
Comment (cfchase) [inline partners/anaconda/base/anaconda-ce-validator-cron.yaml:9]: This looks like it checks once a day, is that enough?

File: partners/anaconda/base/anaconda-ce-imagestream.yaml (hunk 1 above)
Comment (Xaenalt) [reply partners/anaconda/base/anaconda-ce-imagestream.yaml:9]: That can get changed once the changes actually get merged to an official org xD

File: partners/anaconda/base/anaconda-ce-validator-cron.yaml (hunk 2 above)
Comment (Xaenalt) [reply partners/anaconda/base/anaconda-ce-validator-cron.yaml:9]: This is only the logic to disable it in the UI, if the token expires, it's useless to the user as far as using it for Anaconda stuff, I figured this was a good compromise between resource usage and checking often enough

File: partners/anaconda/base/anaconda-ce-imagestream.yaml (hunk 1 above)
Comment (crobby) [reply partners/anaconda/base/anaconda-ce-imagestream.yaml:9]: @Xaenalt Is there a PR for "the changes"?

Comment (crobby) [reply partners/anaconda/base/anaconda-ce-imagestream.yaml:9]: n/m, found it:  https://github.com/red-hat-data-services/s2i-minimal-notebook/pull/1

**Review by crobby — APPROVED** [review]
I tested this along with the corresponding dashboard changes and I think we're all set.

**Pull request merged**